src/devices/cpu/gigatron/gigatron.cpp

```diff
@@ -0,0 +1,129 @@
+// license:BSD-3-Clause
+// copyright-holders:<author_name>
```

Comment:
Remember to fill this in - we can't have uanattributed source files.

Reply:
Who would be the copyright holder?

Reply:
In general the author of the bulk of the material in the file that's eligible for copyright protection ("creative work").  MAMEdev doesn't have a contributor license agreement or a legal entity for copyright assignment.  You could find someone on the team that you'd like to assign copyright to if you really want to, but in most cases people retain copyright on their contributions.